appveyor.yml

```diff
@@ -1,19 +1,19 @@
-os: Visual Studio 2019
 clone_depth: 5
 version: "{branch}.{build}"
+
+image:
+  - Ubuntu
```

Comment:
`Ubuntu` is added here, `Visual Studio 2019` is invariant.
`Ubuntu` build on AppVeyor is failing consistently, eg. https://ci.appveyor.com/project/meowsbits/core-geth/builds/41393408.

```suggestion
```